dependency_licenses.txt

```diff
@@ -14486,6 +14486,32 @@ SOFTWARE.
 
 -----
 
+The following software may be included in this product: esbuild. A copy of the source code may be downloaded from git+https://github.com/evanw/esbuild.git. This software contains the following license and notice below:
```

Comment:
This is from adding `esbuild` as a dev dependency in the conversation transformer.